src/Chainweb/Payload.hs

```diff
@@ -516,6 +556,24 @@ instance IsCasValue TransactionTree where
     type CasKeyType TransactionTree = BlockTransactionsHash
     casKey = _transactionTreeHash
 
+instance ToJSON TransactionTree where
+    toJSON o = object
+        [ "hash" .= _transactionTreeHash o
+        , "tree" .= merkleTreeToJson (_transactionTree o)
+        ]
+
+instance FromJSON TransactionTree where
+    parseJSON = withObject "TransactionTree" $ \o -> TransactionTree
+        <$> o .: "hash"
+        <*> (o .: "tree" >>= merkleTreeFromJson)
+
+merkleTreeToJson :: MerkleTree a -> Value
```

Comment:
`MerkleTree` is defined in the `merkel-log` package which has just a limit set of dependencies. I'd rather avoid to depend on aeson just for the sake of these instances.

Reply:
Since it's used only here, adding a newtype (which would be the idiomatic way) to wrap these function in `ToJSON` and `FromJSON` instances seemed like unneeded overhead. The definition of the encoding and decoding functions would be the same.

Reply:
ok